src/util/lib/String/escapeshellarg.js

```diff
@@ -5,12 +5,12 @@ global.__jymfony = global.__jymfony || {};
 /*
  * Adapted from locutus.io
  */
-global.__jymfony.escapeshellarg = function escapeshellarg (arg) {
+global.__jymfony.escapeshellarg = (arg) => {
     //  discuss at: http://locutus.io/php/escapeshellarg/
     // original by: Felix Geisendoerfer (http://www.debuggable.com/felix)
     // improved by: Brett Zamir (http://brett-zamir.me)
     //   example 1: escapeshellarg("kevin's birthday")
     //   returns 1: "'kevin\\'s birthday'"
 
-    return "'" + arg.replace(/[^\\]'/g, m => m.slice(0, 1) + '\\\'') + "'";
+    return `'${arg.replace(/[^\\]'/g, m => m.slice(0, 1) + '\\\'')}'`;
```

Comment:
In this particular case I find the previous version clearer.
In general i prefer string interpolation of simple expressions (a variable or a property access, also via a getter) and concatenation for more complex expressions like this.